mantis-common/src/main/java/io/mantisrx/common/WorkerPorts.java

```diff
@@ -112,11 +101,12 @@ public int getCustomPort() {
 
     @JsonIgnore
     public List<Integer> getAllPorts() {
-        final List<Integer> allPorts = new ArrayList<>(ports);
+        final List<Integer> allPorts = new ArrayList<>();
```

Comment:
this diff changes the order of the sink port in the list, not sure how this function is used but it might break any code path that relies on sink port being first in list